### ISSUE TITLE
[Feature Request]: Make the option to create a tag more visible

### ISSUE THREAD

**Issue by Bamowen** [TagStudioDev/TagStudio#416]
### Checklist

- [X] I am using an up-to-date version.
- [X] I have read the [documentation](https://github.com/TagStudioDev/TagStudio/blob/main/doc/index.md).
- [X] I have searched existing [issues](https://github.com/TagStudioDev/TagStudio/issues).

### Description

It can be quite a challenge for a first time user to find how to create a tag. At first we're drawn to the "Add Field" button or the right click on a file after opening a library. When we should go to Edit > New Tag or Ctrl+T.

### Solution

Have "Create Tag" instantly accessible.

### Alternatives

This could be an option on by default, that one could chose to hide later.

**Comment by CyanVoxel** [comment]
I agree, this is something that needs to be more readily accessible 👍

**Comment by CyanVoxel** [comment]
Just an update on this, I've currently got a new "Add Tag" button front and center as part of #655. That PR also completely gets rid of the "tag fields" concept in favor of a new automatic and fully customizable tag-based approach:
<img width="279" alt="Screenshot 2025-01-04 at 04 23 43" src="https://github.com/user-attachments/assets/0b92eca5-db8f-4e3e-954b-1b4f3795f073" />
